Assets/scripts/Player.cs

```diff
@@ -11,65 +11,74 @@ public class Card
 	// A number used to determine probablity of card selection. Higher is better.
 	public int Points { get; set; }
 	public Texture2D Icon { get; set; }
+	public AudioClip Sound {get; set; }
 }
 
 public class CardActions
 {
 	public List<Card> Cards = new List<Card>();
-
+	
 	public CardActions()
 	{
 		//AssetDatabase.LoadAssetAtPath("")
 		Cards.Add(new Card() {
 			Title = "Move 3",
+			Sound = Resources.Load<AudioClip>("click-1"),
```

Comment:
do you think it might make sense to load the sound once first, for each sound type, and then assign? not sure if performance matters on this stuff.